src/include/platform/ThreadStackManager.h

```diff
@@ -92,8 +92,10 @@ class ThreadStackManager
     // ===== Members that define the public interface of the ThreadStackManager
 
     CHIP_ERROR InitThreadStack();
+    void DeinitThreadStack();
```

Comment:
Opposite of Init would generally be Shutdown....

Reply:
Renamed to Shutdown.